app/src/main/java/cs5150athletetracking/com/athletetracking/AsyncUploader.java

```diff
@@ -0,0 +1,102 @@
+package cs5150athletetracking.com.athletetracking;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.*;
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.net.URLEncoder;
+import java.util.Iterator;
+
+import org.json.JSONObject;
+
+
+/**
+ * Created by mag94 on 10/19/15.
+ */
+
+public class AsyncUploader extends AsyncTask<JSONObject, String, Integer> implements ResultCallable {
+
+    private static final String TAG = "Http Connection";
+    private static final String urlStr = "http://ec2-54-165-208-160.compute-1.amazonaws.com/php/test_server.php";
+    private static final int HTTP_OK = 200;
+
+    public void success() {
+        //
+    }
+
+    public void failure() {
+        //
+    }
+
+    @Override
+    protected Integer doInBackground(JSONObject... params) {
+        Integer result = 0;
+
+        try {
+            int count = params.length;
+            for (int i = 0; i < count; i++) {
+                JSONObject param = params[i];
+                StringBuilder jsonPost = new StringBuilder();
+                Iterator keys = param.keys();
+
+                // Format LocationJSON argument for the POST method
+                // The format is: "key1=value1&key2=value2"
+                while (keys.hasNext()) {
```

Comment:
I thought that it was just going to be JSON object, this allows you to store different types that aren't necessarily string types. What is the benefit of the other way